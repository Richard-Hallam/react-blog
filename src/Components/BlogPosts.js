import React from "react";
import '../styles/BlogPosts.css';

class BlogPosts extends React.Component{
    render() {
        return 
            <div>
                <ul className="post-list">
                    <li className="post">test</li>
                </ul>
            </div>
    }
}

export default BlogPosts;