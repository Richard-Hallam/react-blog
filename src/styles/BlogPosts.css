.post{
    display: flex;
    list-style-type: none;
    background-color: aqua;
    
}