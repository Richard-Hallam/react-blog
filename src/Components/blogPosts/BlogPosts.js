import React from "react";
import '../../styles/BlogPosts.css';

function BlogPosts(){

        return <div>
            <ul className="post-list">
              <li className="post">
                <div>
                    <h3>Hydroponics control system build 1</h3>
                    <p>THis is just some sample text that is not important right now Hopefully I come up with a better way to do this later on</p>
                </div>
              </li>
            </ul>
        </div>
    }


export default BlogPosts;