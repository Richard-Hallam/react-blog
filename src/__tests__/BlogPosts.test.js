import React from "react";
import { render } from '@testing-library/react';
import BlogPosts from "../Components/blogPosts/BlogPosts";

describe("Projects", () => {
    const { asFragment } = render(<BlogPosts />);

    it("renders correctly", ()=>{
        expect(asFragment()).toMatchSnapshot();
    });


});