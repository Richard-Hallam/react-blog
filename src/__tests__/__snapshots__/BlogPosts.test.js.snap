// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders correctly 1`] = `
<DocumentFragment>
  <div>
    <ul
      class="p-10"
    >
      <li
        class="flex gap-4 flex-col items-center"
      >
        <div
          class="max-w-l rounded overflow-hidden shadow-lg bg-slate-400 max-w-screen-lg"
        >
          <h3
            class="text-2xl font-bold font-mono text-center"
          >
            Title
          </h3>
          <img
            alt="me with a camera"
            src="cameraPicture.jpg"
          />
          <p
            class="text-base font-bold"
          >
            subheading
          </p>
          <p
            class="text-base font-mono"
          >
            body text
          </p>
        </div>
        <div
          class="max-w-l rounded overflow-hidden shadow-lg bg-slate-400 max-w-screen-lg"
        >
          <h3
            class="text-2xl font-bold font-mono text-center"
          >
            Title
          </h3>
          <img
            alt="me with a camera"
            src="cameraPicture.jpg"
          />
          <p
            class="text-base font-bold"
          >
            subheading
          </p>
          <p
            class="text-base font-mono"
          >
            body text
          </p>
        </div>
        <div
          class="max-w-l rounded overflow-hidden shadow-lg bg-slate-400 max-w-screen-lg"
        >
          <h3
            class="text-2xl font-bold font-mono text-center"
          >
            Title
          </h3>
          <img
            alt="me with a camera"
            src="cameraPicture.jpg"
          />
          <p
            class="text-base font-bold"
          >
            subheading
          </p>
          <p
            class="text-base font-mono"
          >
            body text
          </p>
        </div>
        <div
          class="max-w-l rounded overflow-hidden shadow-lg bg-slate-400 max-w-screen-lg"
        >
          <h3
            class="text-2xl font-bold font-mono text-center"
          >
            Title
          </h3>
          <img
            alt="me with a camera"
            src="cameraPicture.jpg"
          />
          <p
            class="text-base font-bold"
          >
            subheading
          </p>
          <p
            class="text-base font-mono"
          >
            body text
          </p>
        </div>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
